libcloud/dns/drivers/gandi.py

```diff
@@ -147,16 +147,20 @@ def delete_zone(self, zone):
         return res.object
 
     def _to_record(self, record, zone):
+        extras = {'ttl': record['ttl']}
+        value = record['value']
+        if record['type'] == 'MX':
+            extras['priority'] = record['value'].split(' ')[0]
+            value = record['value'].split(' ')[1]
```

Comment:
That's for clarifying it.

This does indeed seem like a bug / inconsistency in the Gandi driver. `value` attribute should indeed only contain the value and the priority should be stored in `extra`.